app/services/bookings/gitis/contact.rb

```diff
@@ -158,6 +128,30 @@ def add_school_experience(log_line)
 
         self.dfe_notesforclassroomexperience = "#{dfe_notesforclassroomexperience}#{log_line}\r\n"
       end
+
+    private
+
+      def populate(attrs)
+        super
+
+        self.dfe_channelcreation  = self.class.channel_creation unless dfe_channelcreation.present?
+        self.dfe_Country          = Country.default unless _dfe_country_value.present?
+      end
+
+      def set_email_address_2_if_blank
+        return if emailaddress2.present? || emailaddress1.blank?
+
+        self.emailaddress2 = emailaddress1
+      end
+
+      def set_telephone_2_if_blank(data = {})
+        return if telephone2.present?
+
+        self.telephone2 = data['mobilephone'].presence || \
+          data['address1_telephone1'].presence || \
+          data['telephone1'].presence || \
+          telephone2
```

Comment:
telephone2 wont ever get hit here

Reply:
will if its blank - and this means if there's a blank number in gitis, we leave it as is

Reply:
will if its blank - and this means if there's a blank number in gitis, we leave it as is